script/seed.js

```diff
@@ -77,9 +77,6 @@ async function seed() {
 
   const dbUsers = await User.findAll();
   const userIds = dbUsers.map(user => user.id);
-  const quantities = boatsWithOrdersData.map(_ =>
-    Math.floor(Math.random() * boatsWithOrdersData.length)
```

Comment:
wait what - why is this being removed?

Reply:
Natasha wanted it out.

Reply:
couldn't figure out how to seed database with it, we can re-add if someone does